roles/wireguard/handlers/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# NOTE: Errors are ignored because the kernel module may not be available
+#       after initial installation until after a reboot.
+- name: Restart wg0 service
+  become: true
+  ansible.builtin.systemd:
+    name: wg-quick@wg0.service
```

Comment:
Would prefer to move this into a wireguard_service_name parameter.

Reply:
Done